### PR TITLE
OCD-3549: Add table for new upload

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

**Review by andlar — DISMISSED** [review]

File: ocd-3549.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	deleted bool NOT NULL DEFAULT false,
+	CONSTRAINT certified_product_upload_pk PRIMARY KEY (id),
+	CONSTRAINT certification_body_id_fk FOREIGN KEY (certification_body_id)
+      REFERENCES openchpl.certification_body (certification_body_id) MATCH SIMPLE
+      ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+CREATE TRIGGER certified_product_upload_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.certified_product_upload FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
+CREATE TRIGGER certified_product_upload_timestamp BEFORE UPDATE on openchpl.certified_product_upload FOR EACH ROW EXECUTE PROCEDURE openchpl.update_last_modified_date_column();
+
+--TODO: How to link created certified products to this table? We already have a fk to the old pending table.
```
Comment (tmy1313) [inline ocd-3549.sql]: Does this need to be a discussion with the team?

File: ocd-3549.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+INSERT INTO openchpl.activity_concept (concept, last_modified_user)
+SELECT 'LISTING_UPLOAD', -1
+WHERE
+    NOT EXISTS (
+        SELECT concept FROM openchpl.activity_concept WHERE concept = 'LISTING_UPLOAD'
+    );
```
Comment (tmy1313) [inline ocd-3549.sql]: Extra line break needed.

**Review by tmy1313 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

File: load-pending-changes.sh [hunk 3]
```diff
@@ -33,7 +33,8 @@ then
    helpFunction
 fi
 
-if [ -f ./changes/ocd-????.sql ]
+COUNT_CHANGES_FILES=` find ./changes/ -maxdepth 1 -name 'ocd-????.sql' | wc -l`
```
Comment (andlar) [inline load-pending-changes.sh:36]: I'm curious as to why this was needed

**Review by andlar — APPROVED** [review]

File: load-pending-changes.sh (hunk 3 above)
Comment (kekey1) [reply load-pending-changes.sh:36]: The original problem I had wanted to solve was the script not working when there were 0 files in the changes directory. The line showing as deleted worked fine with 0 or 1 files in the changes directory (as it was in my local environment when I was developing) but when there were > 1 ocd* files in the changes directory it failed with an error that too many arguments were passed in. This was how I fixed that, open to suggestions to improve it.

Comment (andlar) [reply load-pending-changes.sh:36]: I have no problem with the solution, I was just wondering what the problem it was trying to solve was.

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**